src/enums/index.ts

```diff
@@ -19,4 +19,4 @@ export * from './saveErrorReasons'
 
 // external enums
 export { CorpTypeCd, CorpClass } from '@bcrs-shared-components/corp-type-module'
-export { ActionTypes, StaffPaymentOptions } from '@bcrs-shared-components/enums'
```

Comment:
ActionTypes isn't used in this UI, and I'm clearing up confusion around its use anyway, so: cleanup!